proposals/lip-0057.md

```diff
@@ -1,11 +1,11 @@
 ```
 LIP: 0057
-Title: Define state and state transitions of DPoS module
+Title: Define state and state transitions of PoS module
 Author: Maxime Gagnebin <maxime.gagnebin@lightcurve.io>
         Nazar Hussain <nazar@lightcurve.io>
         Mehmet Egemen Albayrak <mehmet.albayrak@lightcurve.io>
         Grigorios Koumoutsos <grigorios.koumoutsos@lightcurve.io>
-Discussions-To: https://research.lisk.com/t/define-state-and-state-transitions-of-dpos-module/320
+Discussions-To: https://research.lisk.com/t/define-state-and-state-transitions-of-pos-module/320
```

Comment:
Here the link redirect to DPoS module. Either we remove it or update it.

Reply:
By accident, the link works correctly though :p. I think it is fine as is.. I am not sure whether we should also inform in the research forum that the terminology has been updated.. Any opinions? (cc. @ricott1 @janhack )